design/sketch.webidl

```diff
@@ -388,17 +388,17 @@ interface GPUColorWriteBits {
 };
 
 dictionary GPUBlendDescriptor {
-    GPUBlendFactor srcFactor;
-    GPUBlendFactor dstFactor;
-    GPUBlendOperation operation;
+    GPUBlendFactor srcFactor = "one";
+    GPUBlendFactor dstFactor = "zero";
+    GPUBlendOperation operation = "add";
```

Comment:
I was going to suggest blending should be off by default, but GPUBlendOperation doesn't seem to have an "none" value. How does one disable blending in WebGPU?

Reply:
IIRC, this is what disabled blending looks like ^^^